docs/source/user-guide/sql/scalar_functions.md

```diff
@@ -507,14 +507,17 @@ tanh(numeric_expression)
 Truncates a number toward zero (at the decimal point).
 
 ```
-trunc(numeric_expression)
+trunc(numeric_expression[, decimal_places])
 ```
 
 #### Arguments
 
 - **numeric_expression**: Numeric expression to operate on.
   Can be a constant, column, or function, and any combination of arithmetic operators.
 
+- **decimal_places**: Optional. The number of decimal places to truncate to.
+  Defaults to 0.
```

Comment:
Doc looks good to me. Just one question. Can `decimal_places` be negative?

Reply:
> 

can not be negative, modify the trunc function just meet the self-defined precision of decimal to '.'

Reply:
TIL turns out that negative precision means "truncate to the left of the decimal place": https://www.postgresqltutorial.com/postgresql-math-functions/postgresql-trunc/#:~:text=If%20the%20precision%20argument%20is,The%20precision%20argument%20is%20optional. 

```
postgres=# select trunc(12345.6789, -2);
 trunc
-------
 12300
(1 row)
```

This is actually what the current datafusion implementation does too so I have added a test and updated the docs